flink-doris-connector/src/main/java/org/apache/doris/flink/catalog/doris/DorisSystem.java

```diff
@@ -180,7 +180,7 @@ public String buildCreateTableDDL(TableSchema schema) {
         //append table comment
         if(!StringUtils.isNullOrWhitespaceOnly(schema.getTableComment())){
             sb.append(" COMMENT '")
-                    .append(schema.getTableComment())
+                    .append(schema.getTableComment().replace("'",""))
```

Comment:
Thanks，you can call `quoteComment` method

Reply:
Thanks,Changed to the quoteComment method.

Reply:
And it has been packaged and tested, no problem